docs/roadmap.md

```diff
@@ -0,0 +1,17 @@
+# Roadmap for CDT
+
+## Summary Milestone 5
+Top priority is clean-up of the code and optimizing for support. 
+- Remove AntlerProj repos from build and archive AntlerProj
+- Vanilla Clang/LLVM
+- Upgrade to llvm 16 
+
+
+## Antler
+We lack the reasources to complete the Antler project and support it going forward. Removing dependancies will simplify the build process and simplify things. 
+
+## Vanilla Clang/LLVM
+Remove the custom extentions to Clang/LLVM. This will enable us to use Vanilla versions of the packages. This will allow us to use the latest, and will lead to improvements in functionality and performance.
```

Comment:
I suggest wording something like `Try eosio extension free llvm and if it works with no issues then remove extensions`